coalib/output/ConfWriter.py

```diff
@@ -1,3 +1,5 @@
+import os
+
 from itertools import chain
```

Comment:
no new line here.
The format should be.
```
Python inbuilt libraries

Third Party libraries

coala libraries
```
itertools and types are inbuilt.

Also note that we keep them alphabetically sorted.